packages/react/src/components/Slider/Slider.js

```diff
@@ -635,6 +635,7 @@ export default class Slider extends PureComponent {
                     aria-valuemin={min}
                     aria-valuenow={value}
                     style={thumbStyle}
+                    aria-labelledby={!ariaLabelInput ? labelId : null}
```

Comment:
Any reason for this being conditional? Testing it out, the violation comes back if `ariaLabelInput` is defined since there isn't anything labelling the element anymore

Reply:
Ahh i see. No no reason I was actually copying the other div. I change it to just labelId